README.md

```diff
@@ -103,7 +99,7 @@ setup and mobile first is a simple personal portfolio project using mobile first
 
 > Describe between 1-3 key features of the application.
 
--Html file with a single header element 
+-Desktop version of my personal portfolio at microverse 
```

Comment:
- [ ] Please 🙏🏽 remove the guidelines presided by `>` for professional documentation.